app/helpers/layout_helper.rb

```diff
@@ -12,5 +12,9 @@ def flash_messages(opts={})
   def show_layout_flash?
     @layout_flash.nil? ? true : @layout_flash
   end
+
+  def event_name_or_invitation_to_guest_user(event)
+    current_user ? event['name'] : 'Want to learn more? Listen in. Next projects review meeting '
```

Comment:
I wonder if this string should be moved into our `config/locales/en.yml` and then we refer to that constant in the cucumber steps rather than the specific output